docs/antrea-network-policy.md

```diff
@@ -477,6 +478,65 @@ spec:
         name: dropMcastUDPTraffic
 ```
 
+#### ACNP for HTTP traffic
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: ClusterNetworkPolicy
+metadata:
+  name: ingress-allow-http-request-to-api-v2
+spec:
+    priority: 5
+    tier: application
+    appliedTo:
+      - podSelector:
+          matchLabels:
+            app: web
+    ingress:
+      - name: allow-http   # Allow inbound HTTP requests to "/api/v2" from Pods with app=client label
+        action: Allow
+        from:
+          - podSelector:
+              matchLabels:
+                app: client
+        l7Protocols:
+          - http:
+              path: "/api/v2"
+      - name: drop-other   # Drop other inbound traffic
+        action: Drop
+```
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: ClusterNetworkPolicy
+metadata:
+  name: egress-allow-http-request
+spec:
+    priority: 5
+    tier: application
+    appliedTo:
+      - podSelector:
+          matchLabels:
+            egress-restriction: http-only
+    egress:
+      - name: allow-dns    # Allow outbound DNS requests
+        action: Allow
+        ports:
+          - protocol: TCP
+            port: 53
+          - protocol: UDP
+            port: 53
+      - name: allow-http   # Allow outbound HTTP requests
+        action: Allow
+        l7Protocols:
+          - http: {}
+      - name: drop-other   # Drop other outbound traffic
+        action: Drop
```

Comment:
clarification question: can this rule actually be "hit"? My understanding is that because there is no L3/L4 criteria in the previous rule (`allow-http`), all traffic that is not `http` will anyway be dropped by Suricata. If my understanding is correct, maybe it would be good to add a comment to that effect here, to avoid potential confusion.